build/files.js

```diff
@@ -63,10 +63,15 @@ const headRegexp = /(^module.exports = \w+;?)/m
       ]
 
     , objectDefinePropertyReplacement = [
-          /Object.defineProperties/
+          /(Object\.defineProperties)/
```

Comment:
I'm not 100% sure this is used anywhere as the previous version didn't actually work